locale/justoverclock-flarum-ext-welcomebox

```diff
@@ -0,0 +1,8 @@
+  forum:
```

Comment:
First line same as with other file, must contain "flarum-ext-welcomebox" and filename must also match that as "flarum-ext-welcomebox.yml".